examples/web-components-kitchen-sink/package.json

```diff
@@ -41,14 +40,15 @@
     "@storybook/core-server": "portal:../../lib/core-server",
     "@storybook/csf-tools": "portal:../../lib/csf-tools",
     "@storybook/manager-webpack4": "portal:../../lib/manager-webpack4",
-    "@storybook/manager-webpack5": "portal:../../lib/manager-webpack5",
     "@storybook/node-logger": "portal:../../lib/node-logger",
     "@storybook/postinstall": "portal:../../lib/postinstall",
     "@storybook/router": "portal:../../lib/router",
     "@storybook/source-loader": "portal:../../lib/source-loader",
     "@storybook/theming": "portal:../../lib/theming",
     "@storybook/ui": "portal:../../lib/ui",
     "@storybook/web-components": "portal:../../app/web-components",
+    "babel-plugin-macros": "3.1.0",
+    "fork-ts-checker-webpack-plugin": "6.2.13",
```

Comment:
I had to set fixed versions because there are some incompatible versions range due to our webpack 4 & 5 mix in the monorepo (should be cleaned in SB 7)